packages/type/package.json

```diff
@@ -77,11 +77,9 @@
   "devDependencies": {
     "@arethetypeswrong/cli": "^0.17.3",
     "@repo/tsconfig": "workspace:*",
+    "@types/node": "^18",
```

Comment:
This package being types, `@types/node` is a required dependency. That's why it's been hoisted to the other packages; as otherwise you get that warning:

> p589c7 → ✘ @inquirer/core@npm:10.1.4 doesn't provide @types/node to @inquirer/type@npm:3.0.2 [61ba9]

We should fix this one, and then we can declare as optional in the peer deps meta field. (e.g required for TS users, optional otherwise.)